python/lsst/ts/cbp/csc.py

```diff
@@ -354,7 +345,10 @@ async def park(self):
         None
 
         """
-        await self._cbp.set_park(1)
+        await self._cbp.set_park()
```

Comment:
I urge you to delete these one-liner methods and call the _cbp methods directly.

This avoids repetition and needless duplication of an API. I'd argue that the CSC contains a "cbp" and the CSC tells it to do things.

Reply:
Please consider moving CBPModel to its own source file. I think that will make it easier for people to read and understand your code.